aiida/cmdline/commands/profile.py

```diff
@@ -29,162 +31,146 @@ def __init__(self):
-                continue
-
-            postgres = Postgres(port=profile.get('AIIDADB_PORT'), interactive=True, quiet=False)
-            postgres.determine_setup()
-            print postgres.dbinfo
-
-            db_name = profile.get('AIIDADB_NAME', '')
-            if not postgres.db_exists(db_name):
-                print("Associated database '{}' does not exist.".format(db_name))
-            elif force or click.confirm("Delete associated database '{}'?\n" \
-                             "WARNING: All data will be lost.".format(db_name)):
-                print("Deleting database '{}'.".format(db_name))
-                postgres.drop_db(db_name)
-
-            user = profile.get('AIIDADB_USER', '')
-            if not postgres.dbuser_exists(user):
-                print("Associated database user '{}' does not exist.".format(user))
-            elif users.count(user) > 1:
-                print("Associated database user '{}' is used by other profiles "\
-                      "and will not be deleted.".format(user))
-            elif force or click.confirm("Delete database user '{}'?".format(user)):
-                print("Deleting user '{}'.".format(user))
-                postgres.drop_dbuser(user)
-
-            repo_uri = profile.get('AIIDADB_REPOSITORY_URI','')
-            repo_path = urlparse(repo_uri).path
-            repo_path = os.path.expanduser(repo_path)
-            if not os.path.isabs(repo_path):
-                print("Associated file repository '{}' does not exist."\
-                      .format(repo_path))
-            elif not os.path.isdir(repo_path):
-                print("Associated file repository '{}' is not a directory."\
-                       .format(repo_path))
-            elif force or click.confirm("Delete associated file repository '{}'?\n" \
-                               "WARNING: All data will be lost.".format(repo_path)):
-                print("Deleting directory '{}'.".format(repo_path))
-                import shutil
-                shutil.rmtree(repo_path)
-
-            if force or click.confirm("Delete configuration for profile '{}'?\n" \
-                             "WARNING: Permanently removes profile from the list of AiiDA profiles."\
-                             .format(profile_to_delete)):
-                print("Deleting configuration for profile '{}'.".format(profile_to_delete))
-                del profiles[profile_to_delete]
-                update_config(confs)
+            profile = profiles[profile_to_delete]
+        except KeyError:
+            echo.echo_info("Profile '{}' does not exist".format(profile_to_delete))
+            continue
+
+        postgres = Postgres(port=profile.get('AIIDADB_PORT'), interactive=True, quiet=False)
+        postgres.dbinfo["user"] = profile.get('AIIDADB_USER')
+        postgres.dbinfo["host"] = profile.get('AIIDADB_HOST')
+        postgres.determine_setup()
+
+        import json
+        echo.echo(json.dumps(postgres.dbinfo, indent=4))
```

Comment:
This debug echo statement should be removed before merging

Reply:
Then I feel it requires some explanation, because this will simply print the dictionary, and the user will have no idea what it means.